test/core.spec.js

```diff
@@ -427,6 +528,26 @@ describe('cli', function () {
       await exec('--prerelease dev')
       getPackageVersion().should.equal('1.1.0-dev.2')
     })
+
+    it('exits with error if an invalid release version is provided', async function () {
+      mock({ bump: 'minor', fs: { 'CHANGELOG.md': '' } })
+      try {
+        await exec('--release-as 10.2')
+        throw new Error('That should not have worked')
```

Comment:
Optional: This would be better as `expect(fn).to.throw` - see doc [here](https://www.chaijs.com/api/bdd/#method_throw). This would have the advantage that there then isn't code in the test (meaning we don't need tests for tests).

Reply:
Good call, I copied this convention from other tests in the file, but I'll update them all.

Reply:
Also updated unrelated tests I could find with the same pattern in other files. I did need to add a dev dependency (`chai-as-promised`) to use `expect().to.be.rejectedAs()` on the async functions.